mpsl: init: Stop triggering sched and PM from ZLI ISRs (#22624)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

Multiprotocol Service Layer libraries
-------------------------------------

>   * Changed implementation of interrupt service rutines wrappers: :c:func:`mpsl_timer0_isr_wrapper`, :c:func:`mpsl_rtc0_isr_wrapper`, :c:func:`mpsl_radio_isr_wrapper`, so that they don't trigger kernel scheduler or don't use any kernel APIs.

Reviewer: ```suggestion
  * Updated the implementation of the following interrupt service routines wrappers:

    * :c:func:`mpsl_timer0_isr_wrapper`
    * :c:func:`mpsl_rtc0_isr_wrapper`
    * :c:func:`mpsl_radio_isr_wrapper`
    
    Now, they do not trigger the kernel scheduler or use any kernel APIs.
```

Author: @peknis thank for quick review. The change was applied.
